build.gradle

```diff
@@ -71,3 +72,7 @@ subprojects {
 
     apply from: '../gradle/publish.gradle'
 }
+
+configure(subprojects.findAll {it.name != 'jaeger-generated'}) {
+   apply plugin: 'net.ltgt.errorprone'
```

Comment:
are we sure we cannot simply configure this plugin to not run against generated code? The docs

https://github.com/tbroyer/gradle-errorprone-plugin#advanced-usage

seem to suggest it's doable, but probably requires a lot of gradle-fu.